package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "theforeman",
+  "version": "1.11.0",
+  "license": "GPL-3.0",
+  "description": "Full lifecycle management system",
```

Comment:
I wonder if maybe we should do something like cockpit does here: https://github.com/cockpit-project/cockpit/blob/master/package.json since we aren't really a node module